src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1567,10 +1569,17 @@ public void updateCounts( final MoleculeCounts moleculeCounts,
 
         final Interval trim = calculateTrim(read.getBaseQualitiesNoCopy(), readCounts);
         if ( trim.size() == 0 ) {
+            readCounts.bumpNLowQualityReads();
             return ReadReport.NULL_REPORT;
         }
 
-        return new ReadReport(read, trim, reference.getRefSeq());
+        final ReadReport readReport = new ReadReport(read, trim, reference.getRefSeq());
+        if ( readReport.getRefCoverage().isEmpty() ) {
+            readCounts.bumpNLowQualityReads();
+        } else {
+            readCounts.bumpNReadsEvaluable();
+        }
+        return readReport;
     }
 
     @VisibleForTesting static Interval calculateTrim( final byte[] quals, final ReadCounts readCounts ) {
```

Comment:
IDE tells me that parameter `readCounts` is not used in this updated version of `calculateTrim`